app/views/docs/functions.phtml

```diff
@@ -79,40 +79,133 @@ $runtimes = $this->getParam('runtimes', []);
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>def main(request, response)
+    return response.send("Hello World!")
+end </code></pre>
+        </div>
+        <b>Installing Dependencies</b>
+        <p>Simply package a Gemfile in your function, Appwrite will handle the rest!</p>
+        </div>
+    </li>
+    <li>
+        <h3>Deno</h3>
+        <div>
+        <div class="ide margin-top-small" data-lang="typescript" data-lang-label="Deno">
+            <pre class="line-numbers"><code class="prism language-typescript" data-prism>export default function(req: any, res: any) {
+    res.send("Hello World!")
+}</code></pre>
+        </div>
+        <b>Installing Dependencies</b>
+        <p>No Special Steps are required for Deno, Appwrite will handle everything!</p>
+        </div>
+    </li>
+    <li>
+        <h3>Dart</h3>
+        <div>
+        <div class="ide margin-top-small" data-lang="dart" data-lang-label="Dart">
+            <pre class="line-numbers"><code class="prism language-dart" data-prism>Future&lt;void&gt; start(Request request, Response response) async {
+    response.send("Hello World!");
+}</code></pre>
+        </div>
+        <b>Installing Dependencies</b>
+        <p>Simply package a pubspec.yaml file in your function, Appwrite will handle the rest!</p>
+        </div>
+    </li>
+    <li>
+    <h3>Rust</h3>
+        <div>
+            <div class="ide margin-top-small" data-lang="rust" data-lang-label="Rust">
+            <pre class="line-numbers">
+<code class="prism language-rust" data-prism>use super::{Response, RequestValue};
+
+pub fn main(req: RequestValue) -> Response {
+    return Response::send("Hello World!".to_string());
+}</code></pre>
+            </div>
+            <b>Installing Dependencies</b>
+            <p>Rust Dependencies are not currently supported.</p>
+        </div>
+    </li>
+    <li>
+    <h3>Swift</h3>
+        <div>
+            <div class="ide margin-top-small" data-lang="swift" data-lang-label="Swift">
+            <pre class="line-numbers">
+<code class="prism language-rust" data-prism>import Foundation
+
+func main(req: RequestValue, res: RequestResponse) -> RequestResponse {
+    return res.send(data: "Hello World!");
+}</code></pre>
+            </div>
+            <b>Installing Dependencies</b>
+            <p>Swift Dependencies are not currently supported.</p>
+        </div>
+    </li>
+</ul>
 
-    <p><b>Dart</b></p>
 
-    <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
-        <pre class="line-numbers"><code class="prism language-bash" data-prism>PUB_CACHE=./.appwrite pub get</code></pre>
-    </div>
+<h2><a href="/docs/functions#build" id="build">Builds</a></h2>
+<p>As of Appwrite version 0.13 deployments needs to be built before they can be activated. This is automatically done after creating a deployment and the time taken can vary depending on the runtime.</p>
```

Comment:
Same problem as before, let's not directly mention any versions. Docs should be universal and talking about current version.

```suggestion
<p>Deployments needs to be built before they can be activated. This is automatically done after creating a deployment and the time taken can vary depending on the runtime.</p>
```